src/pyOpenMS/doc/Manual.tex

```diff
@@ -24,7 +24,7 @@
 \usepackage{tocloft}% http://ctan.org/pkg/tocloft
 \setlength{\cftsubsecnumwidth}{3.5em}% Set length of number width in ToC for \subsection
 
-\title{pyOpenMS }
+\title{pyOpenMS 2.3}
```

Comment:
also add to https://github.com/OpenMS/OpenMS/wiki/Preparation-of-a-new-OpenMS-release